include/RED4ext/Buffer-inl.hpp

```diff
@@ -31,9 +31,9 @@ RED4EXT_INLINE void RED4ext::RawBufferAllocator::Free(void* aData) const
     innerAllocator->Free(allocation);
 }
 
-RED4EXT_INLINE const RED4ext::Memory::IAllocator* RED4ext::RawBufferAllocator::GetAllocator() const
+RED4EXT_INLINE RED4ext::Memory::IAllocator* RED4ext::RawBufferAllocator::GetAllocator() const
 {
-    return reinterpret_cast<const Memory::IAllocator*>(&allocator);
+    return reinterpret_cast<Memory::IAllocator*>(const_cast<uintptr_t*>(&allocator));
```

Comment:
Why the `const` change? I remember that `const_cast` can be undefined behavior.

Reply:
To match the rest of SDK with non-const `IAllocator*` used everywhere. We never write to the pointer (there's no need to), so there shouldn't be undefined behavior.